mesos-docker/build/images/mesos/Dockerfile

```diff
@@ -44,10 +44,10 @@ RUN apt-key adv --keyserver keyserver.ubuntu.com --recv E56151BF && \
     CODENAME=$(lsb_release -cs) && \
     echo $CODENAME && \
     echo $DISTRO && \
-    echo "deb http://repos.mesosphere.io/${DISTRO} trusty main" | tee /etc/apt/sources.list.d/mesosphere.list && \
+    echo "deb http://repos.mesosphere.io/${DISTRO} $CODENAME main" | tee /etc/apt/sources.list.d/mesosphere.list && \
     apt-get -y update && \
     VERSION=$(apt-cache madison mesos | head -1 | awk '{ print $3 }')  && \
-    MARATHON_VERSION=$(apt-cache madison marathon | head -1 | awk '{ print $3 }') && \
     apt-get -y install mesos=$VERSION && \
-    apt-get -y install marathon=$MARATHON_VERSION && \
+    curl -O http://downloads.mesosphere.com/marathon/v1.1.1/marathon-1.1.1.tgz && \
```

Comment:
How about keeping the MARATHON_VERSION variable, but hard coding it to 1.1.1 for now?